keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/TestContainer.kt

```diff
@@ -5,16 +5,17 @@ import org.jooq.SQLDialect.MYSQL
 import org.testcontainers.containers.JdbcDatabaseContainer
 import org.testcontainers.containers.MySQLContainerProvider
 
-internal fun initTestDatabase() = initDatabase(
-  mySQLContainer.authenticatedJdbcUrl,
-  MYSQL
-)
+internal val testDatabase by lazy {
+  initDatabase(mySQLContainer.authenticatedJdbcUrl, MYSQL)
+}
```

Comment:
This gives us a shared `testDatabase` across all the tests that gets lazy initialized